### PR TITLE
Updated README.md

### LINKED ISSUE #1: Update the README
- [ ] <b>Description</b>
    Update the README.
- [ ] <b>Use cases</b>
    1. Up to date documentation on the library.
- [ ] <b>Proposed design</b>
    N/A.
- [ ] <b>Alternative design approaches</b>
    N/A.
- [ ] <b>Workaround</b>
    N/A.
- [ ] <b>Additional information</b>
    N/A.

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
## Description
Closes #1.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [x] **All classes have documentation comments.**
- [x] **All methods have documentation comments.**
- [x] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why.
- [x] **All information to be logged/displayed to the user are internationalized.** If not, explain why. 
- [x] **README.md has been updated if needed.**
- [x] **pom.xml version is set to the latest date** If not, explain why.

## Assignee's Checklist
- [x] **All GitHub action checks have passed.**
- [x] **All reviewers have approved.**
- [x] **Relevant documentation has been updated if needed.**
- [x] **pom.xml version is set to the latest date** If not, explain why.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +58,63 @@ Describe the project.
...
+
+// STDOUT content.
+String stdOut = response1.getOutput(StdType.STDOUT);
+
+// STDERR content.
+String stdErr = response1.getOutput(StdType.STDERR);
+
+...
+```
+For more such examples, checkout [JvmUtilityTest](https://github.com/padaiyal/jCommandLine/tree/main/src/test/java/org/padaiyal/utilities/vaidhiyar/JvmUtilityTest.java)
```
Comment (silvs110) [inline README.md:113]:
```suggestion
For more such examples, checkout [JvmUtilityTest](https://github.com/padaiyal/jVaidhiyar/tree/main/src/test/java/org/padaiyal/utilities/vaidhiyar/JvmUtilityTest.java)
```

File: README.md [hunk 2]
```diff
@@ -58,11 +58,63 @@ Describe the project.
```
Comment (silvs110) [inline README.md:58]: Missing description

**Review by silvs110 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (the-c0d3br34k3r) [reply README.md:58]: Done.

File: README.md (hunk 1 above)
Comment (the-c0d3br34k3r) [reply README.md]: Done.

**Review by silvs110 — APPROVED** [review]

**Pull request merged**